[2024-03-29T05:59:39.752+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2024-03-29T05:59:38.184981+00:00 [queued]>
[2024-03-29T05:59:39.768+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2024-03-29T05:59:38.184981+00:00 [queued]>
[2024-03-29T05:59:39.770+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-03-29T05:59:39.771+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-03-29T05:59:39.772+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-03-29T05:59:39.795+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-03-29 05:59:38.184981+00:00
[2024-03-29T05:59:39.806+0000] {standard_task_runner.py:55} INFO - Started process 7768 to run task
[2024-03-29T05:59:39.813+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2024-03-29T05:59:38.184981+00:00', '--job-id', '51', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpg9p2fnct']
[2024-03-29T05:59:39.814+0000] {standard_task_runner.py:83} INFO - Job 51: Subtask create_table
[2024-03-29T05:59:39.921+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2024-03-29T05:59:38.184981+00:00 [running]> on host 0245ea477832
[2024-03-29T05:59:40.042+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-03-29T05:59:38.184981+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-29T05:59:38.184981+00:00
[2024-03-29T05:59:40.058+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-03-29T05:59:40.070+0000] {sql.py:315} INFO - Running statement: 
			CREATE TABLE IF NOT EXISTS users (
				firstname TEXT NOT NULL,
				lastname TEXT NOT NULL,
				country TEXT NOT NULL,
				username TEXT NOT NULL,
				password TEXT NOT NULL,
				email TEXT NOT NULL
			);
		, parameters: None
[2024-03-29T05:59:40.072+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-03-29T05:59:40.087+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20240329T055938, start_date=20240329T055939, end_date=20240329T055940
[2024-03-29T05:59:40.148+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-03-29T05:59:40.205+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
